app/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/request/deregistration/ZeroRatedExmpApplication.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatsubscription.models.updateVatSubscription.request.deregistration
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{Reads, Writes, _}
+
+case class ZeroRatedExmpApplication(natureOfSupplies: Option[String], repaymentSituation: Boolean,
+                                    zeroRatedSuppliesValue: BigDecimal, estTotalTaxTurnover: BigDecimal )
+
+object ZeroRatedExmpApplication {
+
+  implicit val frontendReads: Reads[ZeroRatedExmpApplication] = (
+    (__ \ "natureOfSupplies").readNullable[String] and
+      (__ \ "repaymentSituation").read[Boolean] and
+      (__ \ "zeroRatedSuppliesValue").read[BigDecimal] and
+      (__ \ "estTotalTaxTurnover").read[BigDecimal]
+    )(ZeroRatedExmpApplication.apply _)
+
+  implicit val desWrites: Writes[ZeroRatedExmpApplication] = (
+    (__ \ "natureOfSupplies").writeNullable[String] and
+      (__ \ "repaymentSituation").write[Boolean] and
+      (__ \ "zeroRatedSuppliesValue").write[BigDecimal] and
+      (__ \ "estTotalTaxTurnover").write[BigDecimal]
+    )(unlift(ZeroRatedExmpApplication.unapply))
```

Comment:
This works absolutely fine, but an easier way to do it is : `implicit val format: Format[ZeroRatedExmpApplication] = Json.format[ZeroRatedExmpApplication]`